src/unwinder/arm/unwinder_arm.cpp

```diff
@@ -154,15 +154,15 @@ typedef struct _ARM_CONTEXT_OFFSETS
 } ARM_CONTEXT_OFFSETS, *PARM_CONTEXT_OFFSETS;
 
 static const ARM_CONTEXT_OFFSETS TrapFrameOffsets =
-{  8, 272, { 248,252,256,260, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, 72 },
-   { 184, 192, 200, 208, 216, 224, 232, 240, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0,
-     ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0, ~0}, 64, 68, 264,
+{  8, 272, { 248,252,256,260, (UINT16)~0, (UINT16)~0, (UINT16)~0, (UINT16)~0, (UINT16)~0, (UINT16)~0, (UINT16)~0, (UINT16)~0, 72 },
```

Comment:
A nit - it would be nicer if instead of all the casts (here and at the places in the code where we compare these to (UINT16)~0), we would define a constant, e.g. NO_OFFSET or OFFSET_NONE or something like that and use it. Would you mind doing that? I'll accept this PR even without such a change as it can be done later, but if you'd be willing to do that, it would be nice.

Reply:
Sure, I will post a followup.